### ISSUE TITLE
Fix macserial Year 2020

### ISSUE THREAD

**Issue by Andrey1970AppleLife** [acidanthera/bugtracker#660]
ERROR: Year 2020 is out of valid modern range [2003, 2012]!

**Comment by Andrey1970AppleLife** [comment]
2all: Need S/N from any Mac made in 2020.

**Comment by vit9696** [comment]
I fixed the error message to correctly print [2010, 2019], but it does not change the overall situation. The issue here, as explained in the [format document](https://github.com/acidanthera/MacInfoPkg/blob/master/macserial/FORMAT.md), is that serials have no more year letters. That's one of the reasons behind the [upcoming serial format change](https://www.macrumors.com/2020/01/06/apple-randomized-serial-numbers-late-2020/).

Unless we made a mistake, Z stands for 2nd half of 2019, and we are out of range for 2020. Given that freshly bought MBP16,1 come with Z in the year letter, the error here is unlikely. For this reason they either have to reuse Z for 2020 or utilise one of the free letters: `ABEIOU`. Since `IO` are not considered valid letters at all, I think `AB` is the very likely option unless they go with digits.

CC @al3xtjames
